docs/applications/containers/how-to-install-docker-images.md

```diff
@@ -0,0 +1,104 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+contributor:
+  name: Jack Wallen
+  link: https://twitter.com/jlwallen
+description: 'An introduction to using Docker, containers, and dockerfiles on your Linode.'
+keywords: 'docker,container,dockerfile'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: Wednesday, June 28, 2017
+modified_by:
+  name: Linode
+published: 'Wednesday, June 28, 2017'
+title: 'How to Install Docker and Pull Images '
+external_resources:
+ - '[Docker Docs](http://docs.docker.com/)'
+ - '[Docker Try it Tutorial](https://www.docker.com/tryit/)'
+ - '[Docker Hub](https://hub.docker.com/)'
+ 
+
+---
+
+
+### Installing Docker and Pulling Images from Docker Hub
+
+This tutorial will walk you through the installation of Docker, and the first steps of pulling down images with the intention of spinning them up as containers. 
+
+
+### Getting Started
+
+In order to install Docker, you must first gain shell access to your Linux deployment. To do this, you must know your Linode IP address. To find that, log into your Linode account and click on the Linodes tab. Within this window, you'll see the **Remote Access tab**; click on that tab to reveal the public IP of your Linode. 
+
+![Remote_Access](/docs/assets/Docker/dockerisntall_a.jpg)
+
+Use the IP to `ssh` into your Linode: 
+
+	ssh root@public_ip
```

Comment:
Your guide assumes root access, but doesn't [specify it](https://www.linode.com/docs/linode-writers-formatting-guide#include-a-note-about-root-or-non-root-users). Did you read that guide at all?

![bitmoji](https://render.bitstrips.com/v2/cpanel/8133962-243506170_5-s4-v1.png?transparent=1&palette=1&width=246)